tests/btb/tuning/test_gp.py

```diff
@@ -118,7 +118,7 @@ def test_predict_x_lt_r_min(self, uniform_mock):
         predicted = tuner.predict(X)
 
         # assert
-        expected = np.array([0.8, 0.9])
+        expected = np.array([[0.8, 0], [0.9, 0]])
```

Comment:
nit: Since you have a first column of values and a second column of 0s, it might make sense to type it as
```python
np.array([[0.8, 0.9], [0.0, 0.0]]).T
```
Especially if you expanded the tests, it would be clearer and less error-prone to add new rows